plugin/src/test/groovy/com/novoda/staticanalysis/internal/detekt/DetektIntegrationTest.groovy

```diff
@@ -14,15 +14,24 @@ import static com.novoda.test.LogsSubject.assertThat
 class DetektIntegrationTest {
 
     private static final String DETEKT_NOT_APPLIED = 'The Detekt plugin is configured but not applied. Please apply the plugin in your build script.'
-    private static final String OUTPUT_NOT_DEFINED = 'Output not defined! To analyze the results, `output` needs to be defined in Detekt profile.'
+    private static final String XML_REPORT_NOT_ENABLED = 'XML report must be enabled. Please make sure to enable "reports.xml" in your Detekt configuration'
 
     @Parameterized.Parameters(name = "{0} with Detekt: {1}")
     static Iterable rules() {
         return [
-                [TestProjectRule.forKotlinProject(), "1.0.0.RC6-2"],
-                [TestProjectRule.forAndroidKotlinProject(), "1.0.0.RC6-2"],
-                [TestProjectRule.forKotlinProject(), "1.0.0.RC8"],
-                [TestProjectRule.forAndroidKotlinProject(), "1.0.0.RC8"],
+                [TestProjectRule.forKotlinProject(), "1.0.0.RC9.2"],
+                [TestProjectRule.forAndroidKotlinProject(), "1.0.0.RC9.2"],
+                [TestProjectRule.forKotlinProject(), "1.0.0-RC10"],
+                [TestProjectRule.forAndroidKotlinProject(), "1.0.0-RC10"],
+                [TestProjectRule.forKotlinProject(), "1.0.0-RC11"],
+                [TestProjectRule.forAndroidKotlinProject(), "1.0.0-RC11"],
+                [TestProjectRule.forKotlinProject(), "1.0.0-RC12"],
+                [TestProjectRule.forAndroidKotlinProject(), "1.0.0-RC12"],
+                // Work fine but requires Gradle 5.x to run
```

Comment:
Added these. Work fine locally for me when I use local installation of Gradle 5.4. 
Unfortunately had to comment our for our 4.10 wrapper.